src/fsm/TransitionsFsmModel.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+from fsm.TrdFsmModel import TrdFsmModel
+
+
+class TransitionsFsmModel(TrdFsmModel):
+    def __init__(self, final_state) -> None:
+        self.final_state = final_state.name if isinstance(final_state, Enum) else final_state
+        self.machine = None
+        self.state = None
+        super(TransitionsFsmModel, self).__init__()
+
+    def init(self, machine):
```

Comment:
Thanks for the recommendation. Please continue recommending, I enjoy learning. 

I will keep the "init" method because it implies the class needs to be initialized, a setter is easier to be overlooked. 

There is a cyclic dependency between TransitionsFsmModel and the machine object, maybe we should find a way to escape this.

Reply:
Then I would rename the `init` method with something like `set_machine` or something to distinguish more between the `__init__` constructor. 

To avoid the cycle behavior why not define the machine inside of the constructor of `TransitionsFsmModel` as `self.machine` like they did in their [quickstart tutorial](https://github.com/pytransitions/transitions#quickstart)? You would then need to add more arguments to `TransitionsFsmModel`.